.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -38,6 +30,20 @@ eg: Bug fix for widget so that it now correctly thrombs the tubas to provide cle
 - [ ] I have added tests to cover my changes
 - [ ] All new and existing tests passed
 
+## Time frame for completing Pull Request review (from creation date):
+- [ ] As soon as possible
+- [ ] In the next two weeks
+- [ ] In the next month
+
+## Focus of Pull Request:
+<!--- Indicate the main focus of the PR. Additional information can be added below in the Hints section -->
+All comments are welcome regardless of which items are checked off.
+- [ ] All of the below (no need to check off the other boxes)
+- [ ] There are no errors or inconsistencies
+- [ ] Everything appears clear and easy to understand
+- [ ] Suggest improvements
+- [ ] Identify areas where functionality and/or documentation is missing
+
```

Comment:
```suggestion
- [ ] Specific aspects to focus on are listed below
- [ ] Specific aspects and known issues _not to_ focus on are listed below
```

Reply:
I'm not sure we need checkboxes for these if they exist in the Hints section below. Perhaps combine them into "Important information in Hints section" or similar?